[16.0][IMP] report_alternative_layout: Adding useful options (#57)
OCA/l10n-japan: report_alternative_layout/readme/CONFIGURE.rst

Go to *Settings > Technical > Reporting > Paper Format*, and open a paper format of your choice, and
update the following fields as necessary.
- Apply Alternative Layout: If selected, the alternative layout will be applied to the reports using this paper format.
- Show Address In Header: If selected, the partner's address will be shown in the report header on every page (only applies to the alternative layout).

Go to *Settings > Technical > Reporting > Reports*, and open a report of your choice, and
update the following fields as necessary.

- Paper Format: Select a paper format with 'Apply Alternative Layout' enabled if you want to use the alternative layout.
- Show Commercial Partner: If selected, the commercial partner of the document partner
  will show in the report output (instead of the document partner).
- Show Remit-to Bank: If selected, the remite-to bank infomation will be shown in the printed report.
- Show Document Number: If selected, the document number will be displayed in the report header, at the top-right corner.

> - Date Field Label: If entered, the label for the date field will be displayed in the report header, at the top-right corner.
> - Date Field: If entered, the date of the document will be displayed in the report header, at the top-right corner.

Reviewer: ```suggestion
- Date Field: If entered, the date of the document will be displayed in the report header, at the top-right corner.
- Date Field Label: The label to display for the date field in the report, if specified. If not set, the field’s name will be used instead.
```

Go to *Settings > Technical > Reporting > Paper Format*, and open a paper format of your choice, and
update the following fields as necessary.

- Apply Alternative Layout: If selected, the alternative layout will be applied to the reports using this paper format.

> - Show Address In Header: If selected, the partner's address will be shown in the report header on every page.(only applies to the alternative layout).

Reviewer: ```suggestion
- Show Address In Header: If selected, the partner's address will be shown in the report header on every page (only applies to the alternative layout).
```

Go to *Settings > Technical > Reporting > Reports*, and open a report of your choice, and
update the following fields as necessary.

- Paper Format: Select a paper format with 'Apply Alternative Layout' enabled if you want to use the alternative layout.
- Show Commercial Partner: If selected, the commercial partner of the document partner
  will show in the report output (instead of the document partner).
- Show Remit-to Bank: If selected, the remite-to bank infomation will be shown in the 
  printed report.

> - Show Document Number: If selected, the dodument number will be shown in the report header.
> - Date filed: If entered, the date will be shown in the report header.

Reviewer: ```suggestion
- Show Document Number: If selected, the document number will be displayed in the report header, at the top-right corner.
- Date Field: If entered, the date of the document will be displayed in the report header, at the top-right corner.
```
